lib/src/components/Store.ts

```diff
@@ -6,10 +6,9 @@ export class Store {
   private propsById: Record<string, any> = {};
   private componentsInstancesById: Record<string, IWrappedComponent> = {};
 
-  setPropsForId(componentId: string, props: any) {
+  updateProps(componentId: string, props: any) {
     this.propsById[componentId] = props;
     const component = this.componentsInstancesById[componentId];
-
```

Comment:
Don't you prefer empty line before if?

Reply:
No. Ideally, functions should not contain more than 4-5 lines. In this case empty lines are not needed as it's easy to understand such a small function.